File: .sdk/tm/js/test/exists.test.js

const { test, describe } = require('node:test')
const { equal } = require('node:assert')


const { MixpanelAnnotationsSDK } = require('..')


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await MixpanelAnnotationsSDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- MixpanelAnnotations SDK exists test

local sdk = require("mixpanel-annotations_sdk")

describe("MixpanelAnnotationsSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
